functions/src/models/users/privilegedInformation.ts

```diff
@@ -1,12 +1,122 @@
 import { firestore } from 'firebase';
+import { IsObject, IsString } from 'class-validator';
+
+import { FirestoreDataConverter } from '@google-cloud/firestore';
 import Timestamp = firestore.Timestamp;
 import DocumentData = firestore.DocumentData;
+import QueryDocumentSnapshot = firestore.QueryDocumentSnapshot;
 
-export interface PrivilegedUserInformation extends DocumentData {
+export interface IPrivilegedUserInformation extends DocumentData {
   address: google.maps.GeocoderResult;
 
   termsAccepted: Timestamp; // acts as a timestamp of when and as a boolean: if accepted it exists.
   termsVersion: string;
   privacyAccepted: Timestamp; // acts as a timestamp of when and as a boolean: if accepted it exists.
   privacyVersion: string;
```

Comment:
Will there be a case when the terms or privacy not being accepted? Are we planning to capture this?

Reply:
no because they can't proceed.